CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 This contains only the most important and/or user-facing changes; for a full changelog, see the commit history.
 
+## [2.5.0](https://github.com/ably/ably-js/tree/2.5.0) (2024-10-31)
```

Comment:
```suggestion
## [2.5.0](https://github.com/ably/ably-js/tree/2.5.0) (2024-11-01)
```

leaving it here for visibility: it's not necessarily `2024-11-01`, just update it to whatever current date is when this PR gets merged and you publish a new version to npm